worlds/oot/Cosmetics.py

```diff
@@ -769,7 +769,7 @@ def patch_instrument(rom, ootworld, symbols):
 
 def patch_cosmetics(ootworld, rom):
     # Use the world's slot seed for cosmetics
-    random.seed(ootworld.multiworld.per_slot_randoms[ootworld.player])
+    random.seed(ootworld.random.getrandbits(16))
```

Comment:
i have no idea. i don't play oot, and know nothing about it, just know that all the other randbits in the world use 8 bits, and this was reported over a month ago and i was getting sick of it still existing.

Reply:
Agreed, however the problem is not the use of the global random in general, but the use of the global random in generate_output, since that runs multi-threaded.